GetStream/stream-chat-swift
# [CIS-1581] Fix showing channel watchers in @ suggestions

### 🔗 Issue Link

CIS-1581

### 🎯 Goal

The goal is to stop showing channel watchers in @ suggestions list. The Andorid shows only channel members for local lookup so that's the inconsistency that needs to be fixed.

### 🛠 Implementation

Exclude `channel.last_active_watchers` from search results.

### 📝 Changes

- `channel.last_active_watchers` are excluded from @ suggestions list

### 🎨 UI Changes

| Before | After |
| --- | --- |
| ![before](https://user-images.githubusercontent.com/12818985/154710059-79e85e6a-8b66-45c5-9107-ee5b342916d9.png) | ![after](https://user-images.githubusercontent.com/12818985/154710090-03b2d79a-23de-4b9a-9e81-68830f980661.png) |

### 🧪 Testing

Manual testing:
1. Run `DemoApp` on device
2. Sign in as any user
3. Create a group channel with some members
4. Switch to browser, open `Dashboard -> Chat -> Moderation`
5. Tap on the created channel so the dashboard admin is a watcher
6. Switch back to the device, open the channel, type `@` to see mention suggestions

See dashboard admin watcher is not suggested.

### ☑️ Contributor Checklist


### 🎉 GIF

![Keep an eye on](https://media.giphy.com/media/oZvduOsUmlsk/giphy.gif)
